zadanie3/src/main/java/gra_w_kosci.java

```diff
@@ -0,0 +1,121 @@
+import java.util.Scanner;
+import java.lang.Exception;
+
+/**
+ * W grze w kości bierze udział dwóch graczy o nazwach: pierwszy, drugi. Każdemu graczowi przypada 5
+ * tur, w każdej turze, gracz wykonuje maksymalnie 10 rzutów dwoma kośćmi jednocześnie. Tury graczy
+ * odbywają się naprzemiennie tzn. grę zaczyna gracz pierwszy, po turze następuje tura gracza
+ * drugiego, następna jest tura gracza pierwszego itd. Możliwe wyniki rzutu jedna kością to jedna z
+ * następujących cyfr {1, 2, 3, 4, 5, 6}. Grę zawsze rozpoczyna gracz ”pierwszy”. Podczas każdej tury
+ * gracz zbiera punkty według zasad opisanych poniżej. Suma punktów każdego gracza przed
+ * rozpoczęciem gry wynosi zero. Wygrywa gracz, który zbierze mniejsza ilość punktów.
+ *
+ * @author Kamil Best
+ */
+class graWKosci {
+    private static final int rzutMin = 1;
+    private static final int rzutMax = 6;
+    private static final int iloscTur = 5;
+
+    /**
+     * Sprawdzenie poprawnosci danych wejsciowych.
+     * Wyrzucenie wyjatku w przypadku blednych danych.
+     *
+     * @param rzut - ilosc oczek wyrzuconych na jednej kostce (1-6)
+     * @throws Exception - wyjatek w przypadku podania jako argument liczby spoza zakresu (1-6)
```

Comment:
Exception to typ dosc generyczny http://www.programcreek.com/2009/02/diagram-for-hierarchy-of-exception-classes/ tu mi bardziej pasuje IllegalArgumentException